gcloud-java-bigquery/src/main/java/com/google/cloud/bigquery/BigQueryException.java

```diff
@@ -44,10 +44,15 @@
   private final BigQueryError error;
 
   public BigQueryException(int code, String message) {
```

Comment:
I see. Then in that case I change my mind and suggests to make all public and make the classes final.